primer.md

```diff
@@ -399,6 +399,31 @@ original event source. And as such, it is expected that CloudEvents attributes
 related to the event producer (such as 'source`and`id`) would be changed from
 the incoming CloudEvent.
 
+There might exist special cases in which it is necessary to create a CloudEvent
```

Comment:
Minor thing, and I don't have a strong opinion but since it popped in my head, should we add a `### Nested CloudEvents` header here so that this stands out a bit?